index.html

```diff
@@ -191,48 +280,116 @@ <h2>Modifications to existing procedures</h2>
+                  <p>If
+                  <var>extension</var>.{{RTCRtpHeaderExtensionCapability/uri}}
+                  or
+                  <var>extension</var>.{{RTCRtpHeaderExtensionCapability/direction}}
+                  is missing, [=exception/throw=] a {{TypeError}} and return
+                  <code>undefined</code>.</p>
                 </li>
                 <li>
-                  Let <var>entry</var> be the entry with the same {{RTCRtpHeaderExtensionCapability/uri}} value in
-                  {{RTCRtpTransceiver/[[HeaderExtensionsToOffer]]}}.
+                  <p>If
+                  <var>extension</var>.{{RTCRtpHeaderExtensionCapability/uri}}
+                  is not equal to the {{RTCRtpHeaderExtensionCapability/uri}} of
+                  the <var>i</var>-th element of
+                  <var>transceiver</var>.{{RTCRtpTransceiver/[[HeaderExtensionsToNegotiate]]}},
+                  [=exception/throw=] an {{InvalidModificationError}} and return
+                  <code>undefined</code>.</p>
                 </li>
-                <li>If <var>ext</var>.{{RTCRtpHeaderExtensionCapability/direction}}  is not {{ RTCRtpTransceiverDirection/"sendrecv"}}, and {{RTCRtpHeaderExtensionParameters/uri}}
-                  indicates a mandatory-to-use attribute that is required to be both
-                  sent and received, [=exception/throw=] an {{InvalidModificationError}}.
+                <li>
+                  <p>If
+                  <var>extension</var>.{{RTCRtpHeaderExtensionCapability/direction}}
+                  is not {{RTCRtpTransceiverDirection/"sendrecv"}} and
+                  {{RTCRtpHeaderExtensionParameters/uri}} indicates a
+                  mandatory-to-use attribute that is required to be both sent
+                  and received, [=exception/throw=] an
+                  {{InvalidModificationError}} and return
+                  <code>undefined</code>.</p>
                 </li>
-                <li>if <var>ext</var>.{{RTCRtpHeaderExtensionCapability/direction}} field is {{RTCRtpTransceiverDirection/"stopped"}}, and {{RTCRtpHeaderExtensionCapability/uri}} indicates
-                  a mandatory-to-implement extension, [=exception/throw=] an {{InvalidModificationError}}.
+                <li>
+                  <p>If
+                  <var>extension</var>.{{RTCRtpHeaderExtensionCapability/direction}}
+                  is {{RTCRtpTransceiverDirection/"stopped"}} and
+                  {{RTCRtpHeaderExtensionCapability/uri}} indicates a
+                  mandatory-to-implement extension, [=exception/throw=] an
+                  {{InvalidModificationError}} and return
+                  <code>undefined</code>.</p>
                 </li>
                 <li>
-                  Set the {{RTCRtpHeaderExtensionCapability/direction}} field of <var>entry</var> to <var>ext</var>'s
-                  {{RTCRtpHeaderExtensionCapability/direction}} field.
+                  <p>If necessary, downgrade
```

Comment:
can you find a better term than "downgrade"?

Reply:
```suggestion
                  <p>If necessary, restrict
```

Reply:
Done